recipes/libsndfile/all/patches/0001-turn-to-warning.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/cmake/SndFileChecks.cmake b/cmake/SndFileChecks.cmake
+index 3528c325..1fdd3a6a 100644
+--- a/cmake/SndFileChecks.cmake
++++ b/cmake/SndFileChecks.cmake
+@@ -234,33 +234,29 @@ endif ()
+ 
+ if (DEFINED ENABLE_STATIC_RUNTIME)
```

Comment:
I would just replace this line by `if (FALSE)`, that's all. msvc runtime is already injected by conan, we don't need the logic inside this block.

Reply:
I would also remove previous block related to MinGW, it's bad to force static link to libgcc and libstdc++ (maybe ask to upstream if there is a strong rational). That should be downstream decision like linking to static or dynamic runtime for msvc.

Reply:
Done